hoodie-client/src/main/java/com/uber/hoodie/table/HoodieMergeOnReadTable.java

```diff
@@ -84,59 +87,13 @@
   // UpsertPartitioner for MergeOnRead table type
   private MergeOnReadUpsertPartitioner mergeOnReadUpsertPartitioner;
 
-  public HoodieMergeOnReadTable(HoodieWriteConfig config, HoodieTableMetaClient metaClient) {
-    super(config, metaClient);
-  }
-
-  /**
-   * UpsertPartitioner for MergeOnRead table type, this allows auto correction of small parquet
-   * files to larger ones without the need for an index in the logFile.
-   */
-  class MergeOnReadUpsertPartitioner extends HoodieCopyOnWriteTable.UpsertPartitioner {
+  // HoodieCompactor to be able to compaction a set of CompactionOperations
+  private HoodieCompactor compactor;
 
-    MergeOnReadUpsertPartitioner(WorkloadProfile profile) {
-      super(profile);
-    }
 
-    @Override
-    protected List<SmallFile> getSmallFiles(String partitionPath) {
```

Comment:
again why the reshuffling.. makes it harder to know whats changed and whats just moved :)